app/Actions/User/Notify.php

```diff
@@ -29,7 +29,7 @@ public function do(Photo $photo): void
 		$users = $users
 			->unique('id', true)
 			->whereNotNull('email')
-			->where('id', '!=', AccessControl::id());
+			->where('id', '!=', Auth::id());
```

Comment:
`Auth::id()` returns `null` or the id.
`Auth::authenticate()->id` will return the id OR will throw an unauthenticated exception.
Yes it is intentional.

One way to _unify_ this would be to move all the `Auth::authenticate()->id` behind `Auth::authenticate()` guards. I.e. know exactly which part of the code requires Authentication and which do not.